### DISCUSSION TITLE
Better support AppImage taskbar icon

### DISCUSSION THREAD

**Discussion by Iceyer** [linuxdeepin/developer-center#3382] [Features Request & Ideas | 特性请求 & 头脑风暴]
Some AppImage could not show correct icon in tasktar, see this.

![image](https://user-images.githubusercontent.com/1117694/184870551-7d8c059f-baf8-4622-941e-01fae15cd515.png)

From: https://bbs.deepin.org/post/241620  

The same issue report by obsidian in ubuntu: 

https://forum.obsidian.md/t/obsidian-app-icon-missing-on-ubuntu-21-10/34954

And @electricface test obsidian on manjaro with gnome，which has the same issue.

So it maybe an bug when found pid with windows info, otherwise we could do an better support for AppImage.

**Comment by Iceyer** [comment]
cc @babyfengfjx

**Comment by BLumia** [comment]
+1 for this, same on Arch Linux with DDE.

**Comment by babyfengfjx** [comment]
Thanks for your  feedback, the issue has been submitted to R & D analysis, such as subsequent optimization will be submitted to the new version as soon as possible .
